city_scrapers/spiders/chi_north_river_mental_health.py

```diff
@@ -0,0 +1,73 @@
+import re
+from dateutil.parser import parse
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthRiverMentalHealthSpider(CityScrapersSpider):
+    name = "chi_north_river_mental_health"
+    agency = "North River Expanded Mental Health Services Program and Governing Commission"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.northriverexpandedmentalhealthservicescommission.org"]
+    start_urls = ["http://www.northriverexpandedmentalhealthservicescommission.org/minutes.html"]
+
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css(".meetings"):
+            meeting = Meeting(
+                title="Governing Commission",   
+                description='',
+                classification="COMMISSION",
+                start="7 pm",
+                end=None,
+                all_day=False,
+                time_notes='',
+                location={
+                    "name": "North River Expanded Mental Health Services Program Governing Commission Office",
+                    "address": "3525 W. Peterson Ave, Unit 306, Chicago, IL 60659",
+                },
+                links=self._parse_links(response),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+    
+
+    def _parse_links(self, response, doc_date, doc_url):
```

Comment:
This function should only accept `self`, item`, and `response` as arguments, since all it needs is the link element and response to make sure the URL is right. The function could go something like this:
```python
def _parse_links(self, item, response):
    return [{"title": "Minutes", "href": response.urljoin(item.attrib["href"])}]
```

The `links` property should be a list of dictionaries with `title` and `href` keys, more info in the docs https://cityscrapers.org/docs/development/#links